test/components/DayPickerSingleDateController_spec.jsx

```diff
@@ -1401,4 +1402,22 @@ describe('DayPickerSingleDateController', () => {
       });
     });
   });
+
+  describe('onOutsideClick', () => {
+    it('should render OutsideClickHandler as it has onOutsideClick prop', () => {
+      const wrapper = shallow(
+        <DayPickerSingleDateController
+          onOutsideClick={() => null}
+        />,
+      );
+      expect(wrapper.find(OutsideClickHandler).length).to.equal(1);
```

Comment:
`expect(wrapper.find(OutsideClickHandler).to.have.lengthOf(1)` has a nicer failure message; nbd tho :-)